### ISSUE TITLE
Train facenet on Caffe

### ISSUE THREAD

**Issue by KaleidoZhouYN** [davidsandberg/facenet#445]
Has anybody trained facenet on Caffe? 

I trained facenet by tensorflow on Ms_celeb_1M dataset and it works well,get 99.4% accuracy on LFW.But when I use Inception_resnet_v2 on Caffe,it didn't converge,the loss is always 10+ in 28K iterations.

I don't know whether it makes some difference with BN,so I'll train the Inception_resnet_v1 again.

If anybody had successfully trained facenet on caffe,please @me,thanks a lot.

**Comment by zhanglonghao1992** [comment]
@KaleidoZhouYN  Can I see your caffe code of facenet？ I want to know how to select triplet samples..3q

**Comment by KaleidoZhouYN** [comment]
@zhanglonghao1992  sorry about that,the embedding and triplet part is written by my friend.I just do the classification part.

**Comment by davidsandberg** [comment]
Closed due to inactivity. Reopen if needed.
